mdpow/forcefields.py

```diff
@@ -56,7 +56,8 @@
 
 import logging
 logger = logging.getLogger("mdpow.forecefields")
-
+if not hasattr(logger, "warning"):
+   logger.warning = logger.warn
```

Comment:
I don't understand why this line was not covered by tests. Can you check if logger in Python 2.7 has warning??

Reply:
Because if we don't need it, we will get rid of it everywhere again. Less code is better!

Reply:
My version of logging with Python 2.7 both has warn and warning.

Why did we need this fudge in the first place then?

If it's not needed please remove it everywhere – less code is better.